account_renumber/test/create_moves.py

```diff
@@ -19,30 +18,21 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
-"""
-Script that creates large amounts of account moves on different days,
-that can be used later for testing the renumber wizard.
-"""
```

Comment:
I would be surprised if pep8 raise an error for that, because docstrings of modules must be at their top level.